sklearn/feature_selection/from_model.py

```diff
@@ -173,6 +173,10 @@ class SelectFromModel(BaseEstimator, SelectorMixin):
         Otherwise train the model using ``fit`` and then ``transform`` to do
         feature selection.
 
+    max_n_features : int, between 0 and number of features, default None.
+        If provided, first n most important features are
+        kept, ignoring threshold parameter.
```

Comment:
Need some help to rewrite this part :) . Could you please tell what should I write?